### DISCUSSION TITLE
custom searchInput placement

### DISCUSSION THREAD

**Discussion by iamravisingh** [lukasbach/react-complex-tree#127] [Q&A]
As the default search input is on top of the tree as an overflow. Is there any way to place the customSearchInput as a first HTML element i.e before the tree there should be searchField? I can use **renderSearchInput**. so, do I need to add some CSS changes to achieve this?

cc: @lukasbach

**Comment by lukasbach** [comment]
The library currently does not provide that out of the box, but you could use a react portal to render it outside of the tree.
